.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```

Comment:
Should I disable CircleCI with this in place?

Reply:
Yeah, I'd say that's fine

Reply:
Should be done